### PR TITLE
Added C++ using bitmasks

### PR DESCRIPTION
**Pull request by jessechoe10**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making the changes.

### CODE REVIEW

File: solutions/bronze/usaco-1276.mdx [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
+id: usaco-1276
+source: USACO Bronze 2023 January
+title: Air Cownditioning II
+author: Jesse Choe
+---
+
```
Comment (SansPapyrus683) [inline solutions/bronze/usaco-1276.mdx:7]: add official stuff, implementation header, time complexity, all that

File: solutions/bronze/usaco-1276.mdx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+		comfort.push_back(c);
+	}
+	for (int i = 0; i < m; i++) {
+		int a, b, p, c;
+		cin >> a >> b >> p >> c;
+		conditioners.push_back({a, b});
+		power.push_back(p);
+		cost.push_back(c);
+	}
+	for(int i = 0; i < (1 << m); i++){
```
Comment (SansPapyrus683) [inline solutions/bronze/usaco-1276.mdx:67]: spaces

File: solutions/bronze/usaco-1276.mdx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+using namespace std;
+
+vector<pair<int, int>> cowlocs;
+vector<int> comfort;
+vector<pair<int, int>> conditioners;
+vector<int> cost;
+vector<int> power;
+int ans = 1e9, n, m;
+
+int check(int subset) {
```
Comment (SansPapyrus683) [inline solutions/bronze/usaco-1276.mdx:23]: what this do

File: solutions/bronze/usaco-1276.mdx [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+vector<pair<int, int>> cowlocs;
+vector<int> comfort;
+vector<pair<int, int>> conditioners;
+vector<int> cost;
+vector<int> power;
+int ans = 1e9, n, m;
```
Comment (SansPapyrus683) [inline solutions/bronze/usaco-1276.mdx:21]: all of these have to be global??

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

File: solutions/bronze/usaco-1276.mdx (hunk 3 above)
Comment (jessechoe10) [reply solutions/bronze/usaco-1276.mdx:23]: it's from the official analysis... i just changed the way subsets are generated

**Pull request closed without merging**